packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/test/feature_counter/views/counter_page_test.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:provider/provider.dart';
+
+import 'package:{{project_name}}/feature_counter/blocs/counter_bloc.dart';
+import 'package:{{project_name}}/feature_counter/views/counter_page.dart';
+
+import '../../helpers/golden_helper.dart';
+import '../../helpers/models/scenario.dart';
+import '../mocks/counter_bloc_mock.dart';
+
+Widget counterPageFactory({
```

Comment:
The Current example is too simple and the code looks good but in real  case scenario the DI of a page could look much more complicated, which will make the test not readable. With that being said could you try to extract the page factory to some other file/folder?

Reply:
It's been moved to a factory file